helmfile/50-applications.yaml

```
@@ -557,7 +561,7 @@ releases:
     app: fluentd
     component: fluentd-forwarder
   inherit: [ template: fluentd-elasticsearch ]
-  installed: {{ .Values.fluentd.enabled }}
+  installed: {{ and .Values.opensearch .Values.fluentd.enabled }}
```

shoudn't this one be like the others?
```suggestion
  installed: {{ and .Values.opensearch.enabled .Values.fluentd.enabled }}
```

---

Thanks!